### ISSUE TITLE
Bundle-Version: 1.0.0.SNAPSHOT must be 1.0.0.qualifier [moved]

### ISSUE THREAD

**Issue by lvca** [orientechnologies/orientdb#806]
This is [Issue 806](http://code.google.com/p/orient/issues/detail?id=806) moved from a Google Code project.
Added by 2012-04-28T23:17:07.000Z by [mrExcept...@gmail.com](http://code.google.com/u/101394746117176466173/).
Please review that bug for more context and additional comments, but update this bug.
 Closed (Fixed).

Original labels: Type-Defect, Priority-Medium, v1.0
### Original description

```
Can you please change the Bundle-Version from 1.0.0.SNAPSHOT to 1.0.0.qualifier? 
The 1.0-SNAPSHOT in your pom has to be preserved this is syntactically correct.  That way maven + tycho can pull in the OrientDB snapshot bundles.

In short:

pom -> 1.0-SNAPSHOT
Bundle Manifest 1.0.0.qualifier
```
